### DISCUSSION TITLE
vim_stuff/use-marks-and-jumps-effectively-in-vim/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#21] [Q&A]
# vim_stuff/use-marks-and-jumps-effectively-in-vim/

Vim, the powerful text editor, offers a multitude of features to enhance your editing efficiency. Among these, marks and jumps stand out as incredibly useful tools for navigation within files and across your editing session. In this article we’ll explore how to effectively use marks and jumps to supercharge your Vim workflow.
Understanding Marks Marks in Vim are like bookmarks in your text. They allow you to quickly jump to specific locations in your file or even across different files.

https://jitesh117.github.io/vim_stuff/use-marks-and-jumps-effectively-in-vim/

**Comment by 0xkarambit** [comment]
I always end up forgetting my marks tho 😂,
btw there is `gi` as well to jump to the last position you were editing and switch to insert mode.
`:help gi`
Nice article.

**Reply by Jitesh117** [reply]
Thanks! Glad you liked it.
